study-builder/studies/rarex/study.conf

```diff
@@ -543,7 +573,7 @@
           ],
           "files": [
             {
-              "filepath": "assets/consent_assent.pdf",
+              "filepath": "pdfs/assent-child-7-17-adults-unable-consent.pdf",
```

Comment:
I don't think this is quite right. The pdf configuration here contains substitutions for `follow_up_yes`, `contact_result_yes`, etc, but the `assent..unable-consent.pdf` file doesn't contain those fields. I think we're missing an updated pdf file for `consent_assent.pdf` and `consent_lar_assent.pdf`. And it seems that the `assent..unable-consent.pdf` file is meant to replace the `rarex-assent-*` pdf configurations a little further down.

Reply:
That's right, actually I think that the PDF file for `consent_lar_assent` and `consent_parental_assent`  should not contain those questions, as it is shown in PDF file itself I think that should be started with `Printed Name of Child/Adolescent/Adult Unable to Consent`. Is it right approach to implement?

as far as I see the `follow_up_yes` & `contact_result_yes` is needed only for 3 other cases, which are `consent-self`, 'consent-lar` and `consent-parental`, not for assents